src/sentry/conf/server.py

```diff
@@ -1954,3 +1962,5 @@ def get_sentry_sdk_config():
 }
 
 SENTRY_USE_UWSGI = True
+
+SENTRY_REPROCESSING_ATTACHMENT_CHUNK_SIZE = 10 ** 7
```

Comment:
Current chunk size in Redis is `2**10` (1MiB), not `10**7` (10MB). I was hoping to remove a special configuration for this, but since that is currently controlled only by Relay, we cannot reuse something existing.

Reply:
I think I asked about this in Slack fwiw and got 10mb as answer. Will change to 1mb

Reply:
It's worth noting that this chunk size differs from the one in filestore, which is at 10MB for attachments nowadays. I don't have strong objections against the larger size, and eventually we'll probably streamline. 

Do you see a good place to define this generically?

Reply:
place as in which file? probably this one...

Reply:
I was thinking about having a config for the generic default cache chunk size. At least, this is a wider configuration than just for reprocessing, as it should be the same with principal ingestion (currently controlled by Relay).